schema/dataKeys.json

```diff
@@ -405,6 +405,10 @@
                   "filePrefix": {
                     "type": "string",
                     "description": "filePrefix to use to select files to add to content gateway"
+                  },
+                  "mirrorOpenshiftPush": {
+                    "type": "boolean",
+                    "description": "whether or not to push to mirror.openshift.com"
```

Comment:
Thanks, can you add `components` too? That seems to be the only other missing one. There are other arrays in this file to copy from (line 26 should be an example)

Reply:
Done